storage/azure/src/server/AzureServerStorage.ts

```diff
@@ -222,15 +222,15 @@ export class AzureServerStorage extends ServerStorage {
 
   public async getDownloadUrl(
     reference: ObjectReference,
-    expiresInSeconds = 3600
+    expiry: { expiresInSeconds?: number; expiresOn?: Date }
```

Comment:
maybe we could have a type there that would ensure (to some extent) only a single property specified during compile time?

Reply:
done